src/frontend/gatsby-config.js

```diff
@@ -24,6 +25,7 @@ module.exports = {
     description: `A tool for tracking blogs in orbit around Seneca's open source involvement`,
     author: `SDDS Students and professors`,
     telescopeUrl,
+    version,
```

Comment:
We probably don't need the changes to this file if we're going to get it via /health.